reference-adaptations/TFN%20Cold%20War/composition.sqe

```diff
@@ -1,19 +1,19 @@
-version=54;
-center[]={2671.946,5,6095.9458};
+version=55;
```

Comment:
Was this a manual edit? I vaguely remember @freghar telling me that we shouldn't update the version number if not necessary (though I'm not 100% if I remember correctly). All our other compositions are still versioned `54` afaik.

Reply:
I didn't touch it. The game's internal sqm version has increased.

Reply:
> 

I manually edited it, feel free to reverse. Didn't know a version number change would cause issues, seems dumb

Reply:
It refers to the internal version of the game engines composition parser, not the version of the composition

Reply:
> > 
> 
> I manually edited it, feel free to reverse. Didn't know a version number change would cause issues, seems dumb

That makes sense because I made a few commits recently and they didn't change the version number. It's possible that the sqm version may have increased in the past two weeks but I think that's unlikely. I'd recommend to reverse this before rollout.